mmv1/templates/terraform/examples/dlp_job_trigger_bigquery_identifying_fields.tf.erb

```diff
@@ -0,0 +1,39 @@
+resource "google_data_loss_prevention_job_trigger" "<%= ctx[:primary_resource_id] %>" {
+	parent = "projects/<%= ctx[:test_env_vars]['project'] %>"
```

Comment:
Nit: we typically use 2-space indentation for our example files instead of tabs

Reply:
Done.